docroot/themes/humsci/su_humsci_theme/patterns/molecules/featured-card/featured-card.html.twig

```diff
@@ -0,0 +1,38 @@
+{% set attributes = attributes.addClass('vertical-card') %}
+
+<div{{ attributes }}>
+
+  <div class="featured-card__container">
+    {% if image %}
```

Comment:
This if statement does not wrap the closing </div> and could leaving trailing </div>s where it shouldnt. Wrap the whole statement or add another if statement around the closing div.

Reply:
Thanks, @sherakama. Fixed.

Reply:
Shouldn't we have some styles for the date when the image is not present.

Reply:
For this pattern, the date is nested within the image <div>. So, if there is no image, there will be no date. Also, the more global vertical (or even horizontal depending on use case) should be used for when there is no date stacking over an image.

Reply:
Ok. just wanted to make sure thats the desired outcome. use case: an event without an image but has a date.... 🤔